certbot-dns-rfc2136/certbot_dns_rfc2136/dns_rfc2136.py

```diff
@@ -71,10 +71,12 @@ def _setup_credentials(self):
         )
 
     def _perform(self, domain, validation_name, validation):
-        self._get_rfc2136_client().add_txt_record(domain, validation_name, validation, self.ttl)
+        self._get_rfc2136_client().add_txt_record(validation_name, validation, self.ttl)
+        dummy = domain
```

Comment:
Instead of this, you can prepend the `domain` argument in the `perform()` with underscore to mark it as unused.

Reply:
Another ways to make pylint ignore a rule for a method would be for example:
```
def _perform(self, domain, validation_name, validation):
    #  pylint: disable=unused-argument
    ...
```
or
```
def _perform(self, domain, validation_name, validation):  #  pylint: disable=unused-argument
    ...
```

but adding the underscore is the correct approach here.